doc/governance/common_member_operations.rst

```diff
@@ -27,7 +27,13 @@ Updating Code Version
 
 For new nodes to be able to join the network, the version of the code they run (as specified by ``enclave.file``) should be first trusted by the consortium of members.
 
-If the version of the code being executed needs to be updated (for example, to support additional endpoints), members can create an ``add_node_code`` proposal, specifying the new code version.
+If the version of the code being executed needs to be updated (for example, to support additional endpoints), members can use the following endpoints depending on the platform:
+
+- SGX
+Create an ``add_node_code`` proposal, specifying the new code version.
+
+- SNP
+If the new code requires a new security policy, create an ``add_security_policy`` proposal, specifying both the raw policy string and the digest of the policy, (SHA-256 of the raw policy)
```

Comment:
I am less and less convinced by the re-use of the code id concept. In C-ACI usage, this code is is the UVM digest, it is totally independent of the CCF enclave code. Only the security policy specifies what code the node is running. In fact, we will validate this UVM identity using signatures, precisely because it can and will change without the application code changing at all.

Reply:
Yes I agree, I think the plan is as part of https://github.com/microsoft/CCF/issues/4193 we verify signatures instead of the identity check then this issue goes away.

We could have a separate, clearly named table in the meantime (something like uvm_measurment) but probably out of scope for this PR?